states/call.go

```diff
@@ -0,0 +1,170 @@
+package states
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/congqixia/birdwatcher/models"
+	"github.com/congqixia/birdwatcher/proto/v2.0/commonpb"
+	"github.com/congqixia/birdwatcher/proto/v2.0/milvuspb"
+	"github.com/congqixia/birdwatcher/proto/v2.0/querypb"
+	"github.com/spf13/cobra"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"google.golang.org/grpc"
+)
+
+func getCallCmd(state State, cli *clientv3.Client, basePath string) *cobra.Command {
+	callCmd := &cobra.Command{
+		Use:   "call",
+		Short: "enter state that could call grpc service of component",
+	}
+
+	callCmd.AddCommand(
+		getCallQueryNodeCmd(state, cli, basePath),
+	)
+
+	return callCmd
+}
+
+func getCallQueryNodeCmd(state State, cli *clientv3.Client, basePath string) *cobra.Command {
+	callQNCmd := &cobra.Command{
+		Use:   "querynode {nodeid}",
```

Comment:
does user know exactly nodeid of each querynode? maybe host address is more simple? or we should offer a cmd like "show available nodeid"?

Reply:
Actually we do. `show session` command will list all online component with component type & id.